core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -248,7 +248,7 @@ public GraphHopper setStoreOnFlush(boolean storeOnFlush) {
      * <pre>
      * {@code
      *   hopper.setProfiles(
-     *     new Profile("my_car").setVehicle("car").setWeighting("shortest"),
+     *     new CustomProfile("my_car").setVehicle("car"),
```

Comment:
maybe the same for fastest below. even though it is still available it's not recommended